app/controllers/works_controller.rb

```diff
@@ -23,6 +23,7 @@ def index
   def new
     group = Group.find_by(code: params[:group_code]) || current_user.default_group
     @work = Work.new(created_by_user_id: current_user.id, group:)
+    @work_decorator = WorkDecorator.new(@work, current_user)
```

Comment:
It does not seem that we actually using this variable in the method, do we need it?